entries/t/theweathernetwork.com.json

```
@@ -0,0 +1,13 @@
+{
+  "The Weather Network": {
+    "domain": "theweathernetwork.com",
+    "contact": {
+      "twitter": "weathernetwork",
+      "facebook": "theweathernetwork",
+      "form": "https://help.theweathernetwork.com/hc/en-us/requests/new"
+    },
+    "categories": [
+      "other"
+    ]
```

```suggestion
    ],
    "regions": [
      "au",
      "ca",
      "de",
      "es",
      "fr",
      "gb",
      "ie",
      "in",
      "us"
    ]
```

Selected from the country switcher